src/algorithms/tracking/OrthogonalTrackSeedingConfig.h

```diff
@@ -46,46 +46,46 @@ namespace eicrecon {
     /// to global settings (more loose) followed by more strict cuts for the central
     /// and forward/backward regions separately.
 
-    float m_maxSeedsPerSpM_filter = 0; // max number of seeds a single middle sp can belong to - 1
-    float m_deltaRMin = 5* Acts::UnitConstants::mm;
-    bool m_seedConfirmation = false;
-    float m_deltaInvHelixDiameter = 0.00003 * 1. / Acts::UnitConstants::mm;
-    float m_impactWeightFactor = 1.;
-    float m_zOriginWeightFactor = 1.;
-    float m_compatSeedWeight = 200.;
-    size_t m_compatSeedLimit = 2;
-    float m_seedWeightIncrement = 0;
+    float maxSeedsPerSpM_filter = 0; // max number of seeds a single middle sp can belong to - 1
+    float deltaRMin = 5* Acts::UnitConstants::mm;
+    bool seedConfirmation = false;
+    float deltaInvHelixDiameter = 0.00003 * 1. / Acts::UnitConstants::mm;
+    float impactWeightFactor = 1.;
+    float zOriginWeightFactor = 1.;
+    float compatSeedWeight = 200.;
+    size_t compatSeedLimit = 2;
+    float seedWeightIncrement = 0;
 
     ///////////////////////////////////////
     /// CENTRAL SEED FILTER PARAMETERS
-    float  m_zMinSeedConf_cent = -250 * Acts::UnitConstants::mm;
-    float  m_zMaxSeedConf_cent = 250 * Acts::UnitConstants::mm;
-    float  m_rMaxSeedConf_cent = 140 * Acts::UnitConstants::mm;
-    size_t m_nTopForLargeR_cent = 1;
-    size_t m_nTopForSmallR_cent = 2;
-    float  m_seedConfMinBottomRadius_cent = 60.0 * Acts::UnitConstants::mm;
-    float  m_seedConfMaxZOrigin_cent = 150.0 * Acts::UnitConstants::mm;
-    float  m_minImpactSeedConf_cent = 1.0 * Acts::UnitConstants::mm;
+    float  zMinSeedConf_cent = -250 * Acts::UnitConstants::mm;
+    float  zMaxSeedConf_cent = 250 * Acts::UnitConstants::mm;
+    float  rMaxSeedConf_cent = 140 * Acts::UnitConstants::mm;
+    size_t nTopForLargeR_cent = 1;
+    size_t nTopForSmallR_cent = 2;
+    float  seedConfMinBottomRadius_cent = 60.0 * Acts::UnitConstants::mm;
+    float  seedConfMaxZOrigin_cent = 150.0 * Acts::UnitConstants::mm;
```

Comment:
@wdconinc what is our convention on snake case vs camel case (I think we use camel case but I can't recall)? Since this PR is largely cosmetic could I ask that you fix the various snake case parameters to camel case in addition to the work you already did to remove the `m_`s?

Reply:
Changes to `camelCase` should now be included.